### ISSUE TITLE
panic on first safescale command when tenants.json contains only one entry

### ISSUE THREAD

**Issue by vplanche-cssi** [CS-SI/SafeScale#93]
With the content of tenants.json file (only one tenant defined), first safescale command will generate a panic in safescaled (catched):

```
{
  "tenants": [
  {
    "client": "outscale",
    "name": "outscale",
    "identity": {
      "AccessKey": "xxxxxxx",
      "SecretKey": "xxxxx",
      "UserID": "xxxxxx"
    },
    "compute": {
      "URL": "outscale.com/api/latest",
      "Region": "eu-west-1",
      "Subregion": "eu-west-1",
      "DefaultVolumeSpeed": "HDD"
    },
    "network": {
      "VPCName": "xxxx",
      "VPCCIDR": "192.168.0.0/16"
    },
    "objectstorage": {
      "Type": "s3",
      "Endpoint": "https://eu-west-1.outscale.com",
      "AccessKey": "xxxx",
      "SecretKey": "xxxxx"
    }
  }
  ]
}
```

Generated error in safescaled log file is:
```ERROR   [2021-05-31 16:43:33.221][error][ 8618] do not known how to handle calltrace type '*runtime.TypeAssertionError': interface conversion: interface {} is map[string]interface {}, not string fail.OnPanic [/go/src/github.com/CS-SI/SafeScale/lib/utils/fail/events.go:164]
goroutine 51 [running]:
github.com/CS-SI/SafeScale/lib/utils/fail.RuntimePanicError(0x101cf3ed0, 0x1a, 0x140009374a8, 0x1, 0x1, 0x14001548000)
	.../go/src/github.com/CS-SI/SafeScale/lib/utils/fail/errors.go:925 +0x94
github.com/CS-SI/SafeScale/lib/utils/fail.OnPanic(0x1020b2020, 0x140014febe0)
	.../go/src/github.com/CS-SI/SafeScale/lib/utils/fail/events.go:164 +0xec
panic(0x1022e1d40, 0x14001540090)
	/opt/homebrew/Cellar/go/1.16.3/libexec/src/runtime/panic.go:971 +0x3f4
github.com/CS-SI/SafeScale/lib/server/resources/operations.CurrentTenant(0x140009376d8)
	.../go/src/github.com/CS-SI/SafeScale/lib/server/resources/operations/tenant.go:51 +0x278
github.com/CS-SI/SafeScale/lib/server/listeners.PrepareJob(0x1026d72c0, 0x1400091a2d0, 0x0, 0x0, 0x101cd61d4, 0x9, 0x0, 0x0, 0x1026fed08, 0x1400091c140)
	.../go/src/github.com/CS-SI/SafeScale/lib/server/listeners/jobmanager.go:51 +0x268
github.com/CS-SI/SafeScale/lib/server/listeners.(*HostListener).List(0x103365620, 0x1026d72c0, 0x1400091a2d0, 0x1400091c140, 0x0, 0x100f65d80, 0x1024abac0)
	.../go/src/github.com/CS-SI/SafeScale/lib/server/listeners/host.go:232 +0x1f0
github.com/CS-SI/SafeScale/lib/protocol._HostService_List_Handler(0x1025796e0, 0x103365620, 0x1026d72c0, 0x1400091a2d0, 0x140009101e0, 0x0, 0x1026d72c0, 0x1400091a2d0, 0x0, 0x0)
	.../go/src/github.com/CS-SI/SafeScale/lib/protocol/safescale.pb.go:12709 +0x1c8
google.golang.org/grpc.(*Server).processUnaryRPC(0x140011ea380, 0x1026f2498, 0x140008b9e00, 0x1400090e100, 0x14001456ae0, 0x103321028, 0x0, 0x0, 0x0)
	.../go/pkg/mod/google.golang.org/grpc@v1.31.0/server.go:1180 +0x3e8
google.golang.org/grpc.(*Server).handleStream(0x140011ea380, 0x1026f2498, 0x140008b9e00, 0x1400090e100, 0x0)
	.../go/pkg/mod/google.golang.org/grpc@v1.31.0/server.go:1503 +0xa50
google.golang.org/grpc.(*Server).serveStreams.func1.2(0x14001468720, 0x140011ea380, 0x1026f2498, 0x140008b9e00, 0x1400090e100)
	.../go/pkg/mod/google.golang.org/grpc@v1.31.0/server.go:843 +0x94
created by google.golang.org/grpc.(*Server).serveStreams.func1
	.../go/pkg/mod/google.golang.org/grpc@v1.31.0/server.go:841 +0x1f8
: runtime panic occurred:
do not known how to handle calltrace type '*runtime.TypeAssertionError': interface conversion: interface {} is map[string]interface {}, not string
```

**Comment by vplanche-cssi** [comment]
Fixed in commit 1660f0e11577d61308483e753fd6fbc6c91c744f

Issue will be closed as soon as a new release is published.

**Comment by vplanche-cssi** [comment]
the receltny published release candidates contain the fix.
